src/scala/io/bazel/rules_scala/scrooge_support/Compiler.scala

```diff
@@ -113,7 +113,7 @@ class Compiler {
     } yield inputFullPath
 
     val rootImporter = FocusedZipImporter.forPaths(None, allJars)
-    allPaths.foreach { inputFullPath =>
+    allPaths.filter(_.endsWith(".thrift")).foreach { inputFullPath =>
```

Comment:
Can we add a comment here that we only should pass thrift files to the compiler. Jars which have manifest or other files in there will cause errors here.